src/test/java/com/coravy/hudson/plugins/github/GithubLinkActionFactoryTest.java

```diff
@@ -0,0 +1,51 @@
+package com.coravy.hudson.plugins.github;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.empty;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import com.coravy.hudson.plugins.github.GithubLinkAction.GithubLinkActionFactory;
+
+import hudson.model.Action;
+
+public class GithubLinkActionFactoryTest {
+    @Rule
+    public JenkinsRule rule = new JenkinsRule();
+
+    private GithubLinkActionFactory factory = new GithubLinkActionFactory();
+
+    private WorkflowJob createExampleJob() throws IOException {
+        return rule.getInstance().createProject(WorkflowJob.class, "example");
+    }
+
+    @Test
+    public void shouldCreateGithubLinkActionForJobWithGithubProjectProperty() throws IOException {
+        WorkflowJob job = createExampleJob();
+        String url = "https://github.com/jenkinsci/github-plugin/";
```

Comment:
its not the test value itself, its just a some data which should remain the same in this test.
Vars should be used if they make some active role in the test
http://xunitpatterns.com/, page 192 (Irrelevant information)